### ISSUE TITLE
a11y_Pagination> Size> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#512]
**Describe the bug**
Pagination>> "1 violations" is appearing for "Size". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/pagination--size'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for Pagination> Size.

**Screenshots**
https://somup.com/c3n2QmZWWU

**Desktop:**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The Issue is Fixed.
Storybook Link : https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/pagination--size
See Scree-cast : http://somup.com/c3jQf0uJtp
Thanks!

**Comment by Uditsingh67** [comment]
Issue does not exist.
See Screen-cast : http://somup.com/c3jvhMUlpY
